### PR TITLE
Create Fundamental Lentil Soup.md

### PR DESCRIPTION
**Pull request by paulcam206**
Add lentil soup recipe

### CODE REVIEW

**Comment by paulcam206** [comment]
Happy to fix any feedback! 👍

File: recipes/Fundamental Lentil Soup.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+category: entree
+---
+
+# Fundamental Lentil Soup
+
+> It's important to nail the fundamentals in any project -- even dinner.
+
+## Ingredients
+
+2 cups green lentils, rinsed
```
Comment (t-eckert) [inline recipes/Fundamental Lentil Soup.md]: These should be styled with `-` to denote a list.

e.g.

```
- 2 cups green lentils, rinsed
- 1/2 onion, chopped medium
```

File: recipes/Fundamental Lentil Soup.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+1 tbsp dry parsley
+1 bay leaf
+1 tbsp olive (or other) oil
+4 cups vegetable broth
+salt
+pepper
+
+## Directions
+
+1. in large sauce pan, saute onions in olive oil on medium heat until clear (5-7 minutes)
```
Comment (t-eckert) [inline recipes/Fundamental Lentil Soup.md]: Though not complete sentences, I think it would look nice to see the first letters capitalized.

**Comment by t-eckert** [comment]
Looks great, Paul! Just left a review! Thank you!

**Comment by paulcam206** [comment]
Should be good now 🎆

**Pull request merged**